inspec.yml

```diff
@@ -0,0 +1,195 @@
+name: cms-ars-5.0-oracle-mysql-8-stig-overlay
+title: Oracle MySQL 8.0 STIG Automated Compliance Validation Profile
+maintainer: MITRE SAF Team
+copyright: MITRE, 2022
+license: Apache-2.0
+summary: "This Security Technical Implementation Guide is published as a tool to improve the security of Department of Defense (DoD) information systems. The requirements are derived from the National Institute of Standards and Technology (NIST) 800-53 and related documents. Comments or proposed revisions to this document should be sent via email to the following address: disa.stig_spt@mail.mil."
```

Comment:
This might also need to change since it's only highlighting DoD STIG. Tbh I think we've been keeping it simple with a line describing the profile

Reply:
should line 4 say 2023 btw?

Reply:
Good catch! @ejaronne ?
Also, sorry but this summary string might be more consistent with our initial overlays: `CMS ARS 5.0 Overlay InSpec Validation Profile for Oracle MySQL 8.0 STIG`